assets/settings/default.json

```diff
@@ -1576,6 +1576,8 @@
   "command_aliases": {},
   // Whether to show user picture in titlebar.
   "show_user_picture": true,
+  // Whether to show onboarding banners in titlebar.
+  "show_onboarding_banner": true,
```

Comment:
Let's either wait for https://github.com/zed-industries/zed/pull/29494 or incorporate the settings approach, as we seem to get more and more title bar settings and we'd better start namespacing them.

Reply:
I've moved this new setting and the existing `show_user_picture` setting into `TitleBarSettings`. If the latter requires nontrivial migration work because folks might have already set it, I can back out the last commit.